packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -145,6 +144,15 @@ export const Settings = ({
     return <AppErrorDialog data-testid="error-dialog" {...{ error }} />;
   }
 
+  const canAddRecoveryPhone =
+    account.recoveryPhone.available &&
+    config.featureFlags?.enableAdding2FABackupPhone === true &&
+    account.totp.verified;
```

Comment:
I think you can remove `totp.verified`, the `account.recoveryPhone.available` endpoint checks that the user has at least 1 recovery code.

Reply:
Oh, good to know!